README.md

```diff
@@ -1 +1,268 @@
-# node-modules-object-lab
\ No newline at end of file
+# Using Object Module
+
+## Objectives
+
+1. Write a simple non-npm module with module.exports = {}
+1. Import your module from another file
+1. Invoke a method from the module
+
+
+## Introduction
+
+In this lab, you'll master how to create, export and import a object module, i.e., a static module.
+
+## Instructions
+
+1. Create a file `module.js` with `module.exports = {}`
+2. Copy customer data object from the snippet below into `module.js` and export the data
+3. Create a file `main.js` 
+4. Import your object in `main.js` with `require()`
+5. Print an email of the second customer from `main.js`
+6. Keep `main.js` and `module.js` in the same folder
+6. Run `$ npm test` to verify
+
+```js
+[
```

Comment:
the data  object provided here is LONG and hard to digest. can we shorten this to just a few properties?